classes/components/PKPStatsJobsTable.inc.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @file components/PKPStatsJobsTable.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPStatsJobsTable
+ * @ingroup classes_components_stats
+ *
+ * @brief A class to prepare the data object for a jobs table UI component
+ */
+
+namespace PKP\components;
+
+use APP\i18n\AppLocale;
+
+class PKPStatsJobsTable
+{
+    /** @var string An ID for this component  */
+    public $id = '';
+
+    /** @var array Configuration for the columns to display in the table */
+    public $tableColumns = [];
+
+    /** @var array Rows to display in the table */
+    public $tableRows = [];
+
+    /** @var null|string Table description */
+    public $description = null;
+
+    /** @var null|string Table label */
+    public $label = null;
+
+    /**
+     * Constructor
+     *
+     * @param $args array Optional arguments
+     */
+    public function __construct(string $id, array $args = [])
+    {
+        AppLocale::requireComponents([
+            LOCALE_COMPONENT_PKP_MANAGER,
+            LOCALE_COMPONENT_APP_MANAGER
+        ]);
+
+        $this->id = $id;
+        $this->init($args);
```

Comment:
If this is the only use of `init`, why not just move that code into the constructor?

Reply:
I'd like to keep those things separated, allowing us to do a better unit testing. Currently we don't have a good code coverage on unit testing, and I'm thinking in work at the next months in cover at least the main classes.